plugins/scaffolder-backend/sample-templates/v1beta2-demo/template.yaml

```diff
@@ -64,9 +64,22 @@ spec:
         url: ./template
         values:
           name: '{{ parameters.name }}'
-          owner: '{{ parameters.owner }}'         
-          system: '{{ parameters.system }}'
-          destination: '{{ parseRepoUrl parameters.repoUrl }}'
+          owner: '{{ parameters.owner }}'
```

Comment:
I just changed that to provide an example of how use here

Reply:
I'm wondering if instead of ammending this template, you could combine this action with the #6016 to make an entirely new template called `Rails Template` which showcases the use of this and the other action instead?

Reply:
Yeh, I did that in #6016 but I  updated here too because I don't know when the #6061 will get merge, but I can remove that until the #6016 pass